### PR TITLE
`MamaLangPlayground.js`  Eitare ektu format korlam mama! 

### PR DESCRIPTION
**Pull request by zhshakib**
`MamaLangPlayground.js` now has a clean structure. It is now more readable and not so messy! Used `lib/mamaLanguage.js` for Language related keywords!

### CODE REVIEW

File: playground_v2/src/lib/mamaLanguage.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const convertToJS = (sourceCode) => {
```
Comment (ahnafyy) [inline playground_v2/src/lib/mamaLanguage.js]: This is a great idea. Now lets take it a step further can you import this into the cli.js?

So we only have one part where we have all the translations for playground and the command line?

Comment (zhshakib) [reply playground_v2/src/lib/mamaLanguage.js]: Thanks for the suggestion @ahnafyy. It's a great joy to work on this repo! I have made some modifications as you instructed! Check them. I hope it's ready to be merged!

**Comment by zhshakib** [comment]
@ahnafyy it's ready now mama, Maybe we need some GitHub action to replace the old playground with the newer one. The newer one is cute.

**Review by sorenblank — APPROVED** [review]
Everything looks good.

**Comment by ahnafyy** [comment]
Will look at this and try to merge if things look good by sunday this week. If not change my name to Ahnaf OPRIO

**Review by ahnafyy — APPROVED** [review]

**Pull request merged**